client/src/components/auth/Login.js

```diff
@@ -1,30 +1,91 @@
-import React, { Component } from 'react';
+import React, { Component } from 'react'
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import { connect } from 'react-redux';
+import {loginUser} from '../../actions/authActions';
 import './Login.css';
 
 class Login extends Component {
+  constructor() {
+    super();
+    this.state = {
+      email: '',
+      password: '',
+      errors: {}
+    }
+    this.onChange = this.onChange.bind(this);
+    this.onSubmit = this.onSubmit.bind(this);
+  }
+
+  onChange(e){
+    this.setState({[e.target.name]: e.target.value});
+  }
+
+  onSubmit(e){
+    e.preventDefault();
+
+    const currentUser = {
+      email: this.state.email,
+      password: this.state.password
+    };
+
+    this.props.loginUser(currentUser);
+
+
+  }
+
+  componentWillReceiveProps(nextProps){
+    if (nextProps.auth.isAuthenticated){
+      this.props.history.push('/dashboard');
+    }
+    if (nextProps.errors){
+      this.setState({errors: nextProps.errors});
+    }
+  }
+
+
   render() {
+    const {errors} = this.state;
+
     return (
-        <div className="login">
-        <div className="container">
-          <div className="row">
-            <div className="col-md-8 m-auto">
-              <h1 className="display-4 text-center">Log In</h1>
-              <p className="lead text-center">Sign in to your Socialgram account</p>
-              <form action="dashboard.html">
-                <div className="form-group">
-                  <input type="email" className="form-control form-control-lg" placeholder="Email Address" name="email" />
-                </div>
-                <div className="form-group">
-                  <input type="password" className="form-control form-control-lg" placeholder="Password" name="password" />
-                </div>
-                <input type="submit" className="btn btn-info btn-block mt-4" />
-              </form>
+      <div className="login">
+    <div className="container">
+      <div className="row">
+        <div className="col-md-8 m-auto">
+          <h1 className="display-4 text-center">Log In</h1>
+          <p className="lead text-center">Sign in to your DevConnector account</p>
```

Comment:
simple text change from "sign in to your DevConnector account" to "Sign in to your Socialgram account"